pkg/virt-launcher/virtwrap/network/bindings_test.go

```diff
@@ -0,0 +1,164 @@
+			}
+			newVMISlirpInterface = func(namespace string, name string) *kubevirtv1.VirtualMachineInstance {
+				vmi := newVMI(namespace, name)
+				vmi.Spec.Domain.Devices.Interfaces = []kubevirtv1.Interface{*kubevirtv1.DefaultSlirpNetworkInterface()}
+				kubevirtv1.SetObjectDefaults_VirtualMachineInstance(vmi)
+				return vmi
+			}
+		)
+		BeforeEach(func() {
+			domain = NewDomainWithSlirpInterface()
+			vmi = newVMISlirpInterface("testnamespace", "testVmName")
+			api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
+			driver = SlirpBindMechanism{iface: &vmi.Spec.Domain.Devices.Interfaces[0], domain: domain}
+		})
+		It("Should create an interface in the qemu command line and remove it from the interfaces", func() {
+			testRunPlug(&driver, "")
+			Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(0))
+			Expect(len(domain.Spec.QEMUCmd.QEMUArg)).To(Equal(2))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[0]).To(Equal(api.Arg{Value: "-device"}))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[1]).To(Equal(api.Arg{Value: "e1000,netdev=default,id=default"}))
+		})
+		It("Should append MAC address to qemu arguments if set", func() {
+			vmi.Spec.Domain.Devices.Interfaces[0].MacAddress = "de-ad-00-00-be-af"
+			testRunPlug(&driver, "")
+			Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(0))
+			Expect(len(domain.Spec.QEMUCmd.QEMUArg)).To(Equal(2))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[0]).To(Equal(api.Arg{Value: "-device"}))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[1]).To(Equal(api.Arg{Value: "e1000,netdev=default,id=default,mac=de-ad-00-00-be-af"}))
+		})
+		It("Should create an interface in the qemu command line, remove it from the interfaces and leave the other interfaces inplace", func() {
+			domain.Spec.Devices.Interfaces = append(domain.Spec.Devices.Interfaces, api.Interface{
+				Model: &api.Model{
+					Type: "virtio",
+				},
+				Type: "bridge",
+				Source: api.InterfaceSource{
+					Bridge: api.DefaultBridgeName,
+				},
+				Alias: api.NewUserDefinedAlias("default"),
+			})
+			testRunPlug(&driver, "")
+			Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(1))
+			Expect(len(domain.Spec.QEMUCmd.QEMUArg)).To(Equal(2))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[0]).To(Equal(api.Arg{Value: "-device"}))
+			Expect(domain.Spec.QEMUCmd.QEMUArg[1]).To(Equal(api.Arg{Value: "e1000,netdev=default,id=default"}))
+		})
+	})
+	Context("when macvtap binding mechanism is selected", func() {
+		var (
+			domain                        *api.Domain
+			driver                        MacvtapBindMechanism
+			vmi                           *kubevirtv1.VirtualMachineInstance
+			ifaceName                     string
+			macvtapInterface              *netlink.GenericLink
+			mtu                           int
+			mac                           net.HardwareAddr
+			cacheFactory                  cache.InterfaceCacheFactory
+			mockNetwork                   *netdriver.MockNetworkHandler
+			NewDomainWithMacvtapInterface = func(macvtapName string) *api.Domain {
```

Comment:
It's moved on the next commit https://github.com/kubevirt/kubevirt/pull/5747/commits/d4a801dee18aaba95fbca019c8bb85c159ab804c

Reply:
And in moving, you choose to define the function in a variable block, then choose to fix them in the next commit. 

I don't follow this argument; imo, if you "just moved" the code, you'd be introducing named functions instead of this.